packages/import/package.json

```diff
@@ -44,7 +44,8 @@
   ],
   "module": "dist/index.js",
   "dependencies": {
-    "@scalar/openapi-parser": "workspace:*"
+    "@scalar/openapi-parser": "workspace:*",
+    "yaml": "^2.4.5"
```

Comment:
I knew it!

Reply:
This is only used to parse yaml from HTML though 😛